### PR TITLE
Add SenseCAP to vendor index

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add SenseCAP to vendor index. Missing from https://github.com/TheThingsNetwork/lorawan-devices/pull/85/files

### CODE REVIEW

File: vendor/index.yaml [hunk 1]
```diff
@@ -1624,3 +1624,6 @@ vendors:
     name: ThingsOfficer Gmbh
     website: https://www.boatofficer.com/
     logo: boatofficer_logo.png
+    
+  - id: sensecap
+    name: SenseCAP
```
Comment (benolayinka) [inline vendor/index.yaml:1629]:
```suggestion
    name: Seeed Technology Co.,Ltd.
```

**Review by neoaggelos — APPROVED** [review]
Make sure to fix CI errors before merging

**Review by johanstokking — CHANGES_REQUESTED** [review]
We already have "Seeed Technology Co., Ltd" in the vendor list.

The vendor ID is not being used, only for references. So the end result is that there are two vendors called "Seeed Technology Co., Ltd" and that is confusing.

**Comment by benolayinka** [comment]
> We already have "Seeed Technology Co., Ltd" in the vendor list.
> 
> The vendor ID is not being used, only for references. So the end result is that there are two vendors called "Seeed Technology Co., Ltd" and that is confusing.

They submitted the profile as sensecap, so I renamed them sensecap. They can fix it later if they need to.

**Review by johanstokking — APPROVED** [review]

**Pull request merged**